### ISSUE TITLE
Disabling vibrate and playSound doesn't work

### ISSUE THREAD

**Issue by VictorAtPL** [zo0r/react-native-push-notification#1362]
# Bug

When using parameters `playSound` and `vibrate` set to `false`, neither has reflection in real life.

## Environment info

```bash
➜ npx react-native info                     
info Fetching system and libraries information...
System:
    OS: Linux 4.15 Ubuntu 18.04.4 LTS (Bionic Beaver)
    CPU: (12) x64 AMD Ryzen 5 2600 Six-Core Processor
    Memory: 2.05 GB / 15.68 GB
    Shell: 5.4.2 - /usr/bin/zsh
  Binaries:
    Node: 12.15.0 - /usr/bin/node
    Yarn: Not Found
    npm: 6.13.4 - /usr/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    Android SDK: Not Found
  IDEs:
    Android Studio: Not Found
  Languages:
    Python: 2.7.17 - /usr/bin/python
  npmPackages:
    @react-native-community/cli: Not Found
    react: ^16.13.1 => 16.13.1 
    react-native: ^0.62.0 => 0.62.0 
  npmGlobalPackages:
    *react-native*: Not Found
```

Library version: 3.1.9

## Steps To Reproduce

1. Invoke `PushNotification.localNotification` function with proper parameters (listed below)
2. Observe that the phone plays sound and vibrates.

Describe what you expected to happen:

1. The phone should not vibrate nor play sound.

## Reproducible sample code

```javascript
    PushNotification.localNotification({
        /* Android Only Properties */
        id: 2,
        // autoCancel: true, // (optional) default: true
        largeIcon: 'ic_launcher', // (optional) default: "ic_launcher"
        smallIcon: 'ic_notification', // (optional) default: "ic_notification" with fallback for "ic_launcher"
        vibrate: false, // (optional) default: true

        /* iOS only properties */
        // alertAction: 'view', // (optional) default: view
        // category: '', // (optional) default: empty string
        // userInfo: {}, // (optional) default: {} (using null throws a JSON value '<null>' error)

        /* iOS and Android properties */
        title: 'My Notification Title', // (optional)
        message: 'My Notification Message', // (required)
        playSound: false,
        priority: 'min',
        // date: new Date(Date.now() + 1),
    });
```

**Comment by VictorAtPL** [comment]
Possible duplicate (for `vibrate`): #1140

**Comment by Dallas62** [comment]
Hi @VictorAtPL 
Thanks for your report,
I will not be able to test this on a real device so if you can test this PR #1042 it would be great!

Change the current version of `react-native-push-notification`:
```
    "react-native-push-notification": "git+https://github.com/fleeteng/react-native-push-notification.git"
````
then run:
```
yarn install
```
**OR**
```
npm install
```
once it's done, you should be able to test the PR, if it's good for you, I will merge the PR.

**Comment by nihiluis** [comment]
I tested this with PushNotification.localNotificationSchedule and I still hear the obnoxious sound.

**Comment by Dallas62** [comment]
Hi, I posted an update on #1140

**Comment by Dallas62** [comment]
Seems to be fix in `3.3.1`.

**Comment by VictorAtPL** [comment]
@Dallas62 
Sorry, I have stopped using this library and wrote my proprietary native code.
